geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/data/HBaseConnectionPool.scala

```diff
@@ -114,7 +115,7 @@ object HBaseConnectionPool extends LazyLogging {
       override def run(): Connection = {
         if (validate) {
           logger.debug("Checking configuration availability")
-          HBaseAdmin.checkHBaseAvailable(conf)
+          HBaseVersions.checkAvailable(conf)
```

Comment:
So... for ingest clients, it kinda sucks when a coprocessor misconfiguration stops ingest.

Is there an obvious option for improving that?  Maybe stopping people from ingesting is worth it;).

Reply:
people complain either way... i think it's configurable now right?